.npmrc

```diff
@@ -0,0 +1 @@
+legacy-peer-deps=true
```

Comment:
this should be removed, it's only needed if vercel isn't configured to use `bun install`